src/Components/Dialogs/GetMessage/GetElement/getElement.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+import style from '../getMessage.module.css';
+
+const getElemen = (context) => context.map((elem, index) => <div key={index.toString()} className={style.item}>{elem}</div>);
```

Comment:
See the comment in getMessage component.
By the way, there is a typo in this line.

Reply:
Same as above